src/Npgsql/NpgsqlTypeLoadingOptions.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+
+namespace Npgsql;
+
+/// <summary>
+/// Options for configuring Npgsql type loading.
+/// </summary>
+sealed class NpgsqlTypeLoadingOptions
+{
+    /// <summary>
+    /// Load table composite type definitions, and not just free-standing composite types.
+    /// </summary>
+    public bool LoadTableComposites { get; init; }
+
+    /// <summary>
+    /// The server is doesn't support full type loading from the PostgreSQL catalogs, support the basic set
+    /// of types via information hardcoded inside Npgsql.
```

Comment:
We should probably rewrite this sentence...

Reply:
Good catch, thansk